modules/web-cli/assets/js/utils/helpers.js

```diff
@@ -1,35 +1,5 @@
-// TODO: Copied from `core/common/assets/js/utils/helpers.js` and modified into static functions.
 export default class Helpers {
-	static softDeprecated( name, version, replacement ) {
```

Comment:
How bc Works?

Reply:
No BC Needed since all the calls to Helpers were via import. And those are not externals.